stdlib/Test/test/runtests.jl

```diff
@@ -183,18 +183,18 @@ let fails = @testset NoThrowTestSet begin
     end
 
     let str = sprint(show, fails[11])
-        @test occursin("Expression: isapprox(1 / 2, 2 / 1, atol=1 / 1)", str)
-        @test occursin("Evaluated: isapprox(0.5, 2.0; atol=1.0)", str)
+        @test occursin("Expression: isapprox(1 / 2, 2 / 1, atol = 1 / 1)", str)
+        @test occursin("Evaluated: isapprox(0.5, 2.0; atol = 1.0)", str)
     end
 
     let str = sprint(show, fails[12])
-        @test occursin("Expression: isapprox(1 - 2, 2 - 1; atol=1 - 1)", str)
-        @test occursin("Evaluated: isapprox(-1, 1; atol=0)", str)
+        @test occursin("Expression: isapprox(1 - 2, 2 - 1; atol = 1 - 1)", str)
+        @test occursin("Evaluated: isapprox(-1, 1; atol = 0)", str)
     end
 
     let str = sprint(show, fails[13])
         @test occursin("Expression: isapprox(1, 2; k...)", str)
-        @test occursin("Evaluated: isapprox(1, 2; atol=0, nans=true)", str)
+        @test occursin("Evaluated: isapprox(1, 2; atol = 0, nans = true)", str)
```

Comment:
:(

Reply:
I find the spaces more readable, and I know at least some others agree. It also simplifies the code by not needing to special-case identifiers ending in `!`.

Reply:
Matches https://github.com/domluna/JuliaFormatter.jl as well as BlueStyle, I believe.

Reply:
No, both Blue and YASGuide say no spaces around `=` for keyword arguments. AFAIK JuliaFormatter is the only one that does it.

Reply:
I guess it's one thing if you prefer no spaces around `=`, but I don't see why spaces would be considered good for assignments but bad for keyword arguments. Interestingly, https://www.python.org/dev/peps/pep-0008/ agrees with you. It recommends this style:
```
function f(; x::Int = 0, y=1)
```
i.e. using spaces only when there is a type declaration. That seems pretty fussy to me though. Why not just always allow spaces?

Reply:
Though not stated explicitly, it's also what JuliaStats uses, modeled after what's used in much of Base.

Reply:
> I don't see why spaces would be considered good for assignments but bad for keyword arguments.

It provides visual distinction between the two, since they aren't the same.

Reply:
What about `=>` for computed keyword arguments?

Reply:
Most guides (and code) I've seen use space around `=>` regardless of where it occurs, since it always means the same thing.

Reply:
You know, I'm not sure this is the right place to discuss this. The expression printer is not a style guide or code formatting tool, so I'm not sure it matters.